src/components/editor/editor.tsx

```diff
@@ -238,6 +245,10 @@ export class Editor implements ComponentInterface {
       return undefined;
     }
 
-    return <stylo-toolbar containerRef={this.containerRef}></stylo-toolbar>;
+    const style: Record<string, string> = {
```

Comment:
if style has to be set, I would rather move it to its corresponding scss file

Reply:
okay, I've never used stencil/react so don't know exactly how to do